### PR TITLE
resource_control: add a example to reset user resource group to default value

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

File: sql-statements/sql-statement-alter-user.md [hunk 1]
```diff
@@ -175,6 +179,44 @@ ALTER USER 'newuser' RESOURCE GROUP rg1;
...
+
+```
++---------+---------------------------------------------------+
+| USER    | JSON_EXTRACT(User_attributes, "$.resource_group") |
++---------+---------------------------------------------------+
+| newuser | ""                                                |
++---------+---------------------------------------------------+
+1 row in set (0.02 sec)
+```
+
```
Comment (hfxsd) [inline sql-statements/sql-statement-alter-user.md:218]:
```suggestion
```

File: sql-statements/sql-statement-alter-user.md [hunk 2]
```diff
@@ -175,6 +179,43 @@ ALTER USER 'newuser' RESOURCE GROUP rg1;
 Query OK, 0 rows affected (0.02 sec)
 ```
 
+查看当前用户绑定的资源组：
```
Comment (hfxsd) [inline sql-statements/sql-statement-alter-user.md:182]:
```suggestion
查看当前用户绑定的资源组：

```

File: sql-statements/sql-statement-alter-user.md [hunk 3]
```diff
@@ -175,6 +179,44 @@ ALTER USER 'newuser' RESOURCE GROUP rg1;
...
+```
++---------+---------------------------------------------------+
+| USER    | JSON_EXTRACT(User_attributes, "$.resource_group") |
++---------+---------------------------------------------------+
+| newuser | "rg1"                                             |
++---------+---------------------------------------------------+
+1 row in set (0.02 sec)
+```
+
+取消用户绑定的资源组（将用户绑定的资源组设置为空）：
```
Comment (hfxsd) [inline sql-statements/sql-statement-alter-user.md:197]: 取消绑定后，用户会被分配到 default 资源组吗？
```suggestion
取消用户绑定的资源组，即将用户绑定的资源组设置为空：
```

Comment (glorv) [reply sql-statements/sql-statement-alter-user.md:197]: 是的

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-alter-user.md [hunk 4]
```diff
@@ -175,6 +179,44 @@ ALTER USER 'newuser' RESOURCE GROUP rg1;
 Query OK, 0 rows affected (0.02 sec)
 ```
 
+查看当前用户绑定的资源组：
+
+```sql
+SELECT USER, JSON_EXTRACT(User_attributes, "$.resource_group") from mysql.user WHERE user = "rg1";
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-user.md:185]:
```suggestion
SELECT USER, JSON_EXTRACT(User_attributes, "$.resource_group") FROM mysql.user WHERE user = "rg1";
```

File: sql-statements/sql-statement-alter-user.md [hunk 5]
```diff
@@ -62,6 +62,8 @@ SHOW CREATE USER 'newuser';
 1 row in set (0.00 sec)
 ```
 
+### 修改用户基本信息
+
 {{< copyable "sql" >}}
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-user.md:67]:
```suggestion
修改用户 `newuser` 的密码：
```

Comment (Oreoxmt) [reply sql-statements/sql-statement-alter-user.md:67]: L92 https://github.com/pingcap/docs-cn/blob/a682ffbb42efdddd531a2d453be230639acd00ed/sql-statements/sql-statement-alter-user.md?plain=1#L92-L96
修改为：
````markdown
锁定用户 `newuser`：

```sql 
ALTER USER 'newuser' ACCOUNT LOCK; 
``` 
````

File: sql-statements/sql-statement-alter-user.md [hunk 6]
```diff
@@ -175,6 +179,44 @@ ALTER USER 'newuser' RESOURCE GROUP rg1;
...
+```
+
+```
+Query OK, 0 rows affected (0.02 sec)
+```
+
+```sql
+SELECT USER, JSON_EXTRACT(User_attributes, "$.resource_group") from mysql.user WHERE user = "rg1";
+```
+
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-user.md:210]:
````suggestion
取消用户绑定的资源组，即将用户绑定的资源组设置为空。取消后，用户会绑定到 `default` 资源组。

```sql
ALTER USER 'newuser' RESOURCE GROUP ``;
SELECT USER, JSON_EXTRACT(User_attributes, "$.resource_group") FROM mysql.user WHERE user = "rg1";
```

````

File: sql-statements/sql-statement-alter-user.md (hunk 5 above)
Comment (glorv) [reply sql-statements/sql-statement-alter-user.md:67]: 目前这里是把 alter user 的示例分为两类，修改基本信息和修改绑定的资源组。其中修改基本信息实际包含了多个不同的示例，包括修改属性、注释、密码更新策略等，所以整个用了小标题「修改基本信息」。

File: sql-statements/sql-statement-alter-user.md (hunk 4 above)
Comment (glorv) [reply sql-statements/sql-statement-alter-user.md:185]: image
奇怪，我这里 v7.0 的 playground 没问题，不过 where 条件确实写错了，应该是 `WHERE user = 'newuser'`

**Pull request merged**